lnrpc/routerrpc/router.proto

```diff
@@ -213,6 +213,43 @@ message ResetMissionControlRequest{}
 
 message ResetMissionControlResponse{}
 
+message QueryMissionControlRequest {}
+
+/// QueryMissionControlResponse contains mission control state per node.
+message QueryMissionControlResponse {
+    repeated NodeHistory nodes = 1;
+}
+
+/// NodeHistory contains the mission control state for a particular node.
+message NodeHistory {
+    /// Node pubkey
+    bytes pubkey = 1;
+
+    /// Time stamp of last failure. Set to zero if no failure happened yet.
+    int64 last_fail_time = 2;
+
+    /// Estimation of success probability for channels not in the channel array.
+    float other_chan_success_prob = 3;
+
+    /// Historical information of particular channels.
+    repeated ChannelHistory channels = 4;
+}
+
+/// NodeHistory contains the mission control state for a particular channel.
+message ChannelHistory {
+    /// Short channel id
+    uint64 channel_id = 1;
+    
+    /// Time stamp of last failure.
+    int64 last_fail_time = 2;
+
+    /// Minimum penalization amount.
+    int64 min_penalize_amt = 3;
```

Comment:
should have `sat` or `msat` suffix

Reply:
Do we have a final guideline on that? We currently use the suffix pretty randomly

Reply:
for amounts the standard is to always have the suffix, so it's clear w/o needing to turn to documentation. some of them do not have it since they were preexisting, and it would break the API (though not the wire serialization)

Reply:
`sat` added